manifests/server/dependencies.pp

```diff
@@ -22,9 +22,7 @@
     manage_repo => true,
   }
 
-  class { 'nodejs':
-    repo_url_suffix => '14.x',
-  }
+  class { 'nodejs': }
```

Comment:
now that we don't pass any parameters, you can just use `include nodejs`

Reply:
good point, will add that